catalog/ui/src/app/Catalog/Catalog.tsx

```diff
@@ -294,14 +296,23 @@ const Catalog: React.FunctionComponent = () => {
     }
   }, [fetchState]);
 
-  if (showRequestForm) {
+  if (showRequestForm || showWorkshopForm) {
     if (openCatalogItem) {
-      return (
-        <CatalogItemRequestForm
-          catalogItem={openCatalogItem}
-          onCancel={onRequestCancel}
-        />
-      );
+      if (showRequestForm) {
+        return (
+          <CatalogItemRequestForm
+            catalogItem={openCatalogItem}
+            onCancel={onRequestCancel}
+          />
+        );
+      } else {
```

Comment:
this line is could be removed

Reply:
Can you explain?

Reply:
Sure, 
Sorry i meant the else condition could be removed, I wasnt to clear in my previous comment.
So there is a return inside the if, which means it will stop and exit this context. Therefore, if the first condition is not applied you want to execute the else (which doesn't matter if is inside the else statement or not, because will be executed regardless).

In code these tho blocks does the same:

```
function() {
  if (condition) {
    return something;
  } else {
    return somethingElse;
   }
}
```

```
function() {
  if (condition) {
    return something;
  }
  return somethingElse;
}
```

Reply:
I like it. I'll change it.